cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -453,6 +461,112 @@ private int runTestSuit(Target target, Package currentPackage, JBallerinaBackend
         return proc.waitFor();
     }
 
+    private int runTestSuiteWithNativeImage(Package currentPackage, JBallerinaBackend jBallerinaBackend, Target target)
```

Comment:
Shall we move this to a new task?

Reply:
Moved the implmentation of the runtesttask